### PR TITLE
Add support for --track-widget-creation back to ios build rules.

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

**Comment by jacob314** [comment]
Fixes https://github.com/flutter/flutter/issues/18036

File: packages/flutter_tools/bin/xcode_backend.sh [hunk 1]
```diff
@@ -109,6 +109,12 @@ BuildApp() {
     preview_dart_2_flag="--no-preview-dart-2"
   fi
 
+  local track_widget_creation_flag=""
+  if [[ -n "$TRACK_WIDGET_CREATION" ]]; then
+    track_widget_creation_flag="--track-widget-creation"
+  fi
+
+
```
Comment (cbracken) [inline packages/flutter_tools/bin/xcode_backend.sh]: nit: kill the extra blank line

**Review by cbracken — APPROVED** [review]

**Comment by cbracken** [comment]
lgtm

**Comment by devoncarew** [comment]
@jacob314, we may also want to change some of the reload device lab tests to use `--track-widget-creation` when they flutter run, in order to make sure that code path has more testing.

**Pull request merged**